pytrip/res/point.py

```diff
@@ -33,15 +33,23 @@ def get_basis_from_angles(gantry, couch):
 
 
 def angles_from_trip(gantry, couch):
-    gantry += 90
-    couch = -(couch + 90)
-    return gantry, couch
+    """
+    TODO
+    :param gantry:
+    :param couch:
+    :return:
+    """
+    return gantry + 90.0, -couch - 90.0
 
 
 def angles_to_trip(gantry, couch):
-    gantry -= 90
-    couch = -couch - 90
-    return gantry, couch
+    """
+    TODO
+    :param gantry:
+    :param couch:
+    :return:
+    """
+    return gantry - 90.0, -couch - 90.0
```

Comment:
It would be nice if there was some comment, even with some link to a page that tells why you use those values to change type of angles